dataland-frontend/src/components/forms/CreateEuTaxonomyForFinancials.vue

```diff
@@ -516,6 +519,37 @@ export default defineComponent({
         .catch((e) => console.log(e));
     },
 
+    /**
+     * Parses received financial service types and eligibility KPIs data
+     * @param receivedFormInputsModelData Received data
+     * @returns An object with financial service types and eligibility KPIs which can be directly merged into the formInputsModel, which will then populate expected KPI section fields
+     */
+    extractEligibilityKpis(receivedFormInputsModelData: ObjectType) {
```

Comment:
We are not only extracting eligibility kpis but also other financial service specific kpis. I'd be fine with just dropping 'eligibility' from the method's name.

Reply:
Done.